### PR TITLE
New Site Release

### LINKED ISSUE #99: Make All Pages Translatable
## Description

Right now, some pages haven't really been made translatable via i18n, so we'll need to eventually start moving strings into the i18n for those pages. Note that we won't be translating blog posts or press links, just page content.

### LINKED ISSUE #102: Spacing issue on 'What is a release candidate?' faq entry
## Describe the bug

In the FAQ entry 'What is a release candidate?' there appears to be a missing space

## To Reproduce

1. Go to https://rockylinux.org/faq/
3. Scroll down to 'What is a release candidate?'

## Expected behavior

There should be a space.

## Screenshots

image

## Device Information

**Desktop (please complete the following information):**

- OS: Pop_OS!
- Browser: Firefox
- Version: 88

### LINKED ISSUE #108: Missing variable for translation
## Describe the bug

On the footer of the website you have 3 links that aren't translated. After check on the translation.json and the website code, these links name are hardcoded into the website.
 
- `Community Charter`
- `Organizational Structure`
- `Privacy Policy`

## To Reproduce

1. Go the https://rockylinux.org/ 
2. Select any language 
3. Go to the footer section of the page
4. See that it's still write in english.

## Expected behavior

These links should be translated

### LINKED ISSUE #113: Links have the prefix 'links.' based on chosen language
## Describe the bug
Based on the chosen language, some Links on the Homepage have a `links.` prefix, e.g.: `links.download` in the Download Button or `links.partners` in the footer. This is somehow not the case for the french and english version of the website.

## To Reproduce

1. Go to 'https://rockylinux.org/'
2. Change the language on the bottom to any except french or english
3. Look at the Download Button and the footer

## Expected behavior
Links should be displayed without the `links.` prefix, e.g.: `links.download` should be `Download`.

## Screenshots
Homepage Download Buttons Error
Homepage Footer Links Error

## Device Information

**Desktop (please complete the following information):**

- OS: Windows 10
- Browser: firefox
- Version: 88.0.1

**Smartphone (please complete the following information):**

- Device: iPhone SE 2020
- OS: [e.g. iOS 14.5.1
- Browser: firefox daylight
- Version: 33.1

### LINKED ISSUE #114: Add torrent links to downloads page
Torrents are available in the downloads directories (ex https://download.rockylinux.org/pub/rocky/8.3/isos/x86_64/Rocky-8.3-x86_64-dvd1.torrent) but not linked on the [downloads page](https://rockylinux.org/download)

### LINKED ISSUE #117: Add export compliance blurb to Downloads page
Please add the following blurb to the Downloads page:

# Export Compliance/Customs Information

By downloading Rocky Linux software, you acknowledge that you understand all of the following: Rocky Linux software and technical information may be subject to the U.S. Export Administration Regulations (the “EAR”) and other U.S. and foreign laws and may not be exported, re-exported or transferred (a) to a prohibited destination country under the EAR or U.S. sanctions regulations (currently Cuba, Iran, North Korea, Sudan, Syria, and the Crimea Region of Ukraine, subject to change as posted by the United States government); (b) to any prohibited destination or to any end user who has been prohibited from participating in U.S. export transactions by any federal agency of the U.S. government; or (c) for use in connection with the design, development or production of nuclear, chemical or biological weapons, or rocket systems, space launch vehicles, or sounding rockets, or unmanned air vehicle systems. You may not download Rocky Linux software or technical information if you are located in one of these countries or otherwise subject to these restrictions. You may not provide Rocky Linux software or technical information to individuals or entities located in one of these countries or otherwise subject to these restrictions. You are also responsible for compliance with foreign law requirements applicable to the import, export and use of Rocky Linux software and technical information.
Rocky Linux software in source code and binary code form are publicly available and are not subject to the EAR in accordance with §742.15(b).

### LINKED ISSUE #118: Add 280c wording to Fosshost on Partners page
Please add the following wording:

> Fosshost (UK CIC #13356530) offers free enterprise grade internet hosting and managed services exclusively for the FOSS community. We're on a mission to empower and support every free and open-source software project. To go further, together. Our work never stops.  Apply today!

### PR DESCRIPTION
**Pull request by FoggyMtnDrifter**
Enhancements:
- Start using i18n namespace (@ofaruksahintr)
- Added DVD torrent links to download page (closes #114)
- Added export compliance blurb to download page (closes #117)
- Added Fosshost text blurb to partners page (closes #118)

Other fixes:
- closes #113 
- closes #108 
- closes #102 
- closes #99

### CODE REVIEW

**Comment by hbjydev** [comment]
#### 📝Changed routes:


| [**/download**](https://rockylinux-gnv9d3fdu-resforg.vercel.app/download) | [**/faq**](https://rockylinux-gnv9d3fdu-resforg.vercel.app/faq) |
|:-:|:-:|
|<a href="https://rockylinux-gnv9d3fdu-resforg.vercel.app/download">Screenshot of /download</a><br /><sup><a href="https://images.weserv.nl?url=https://iad.microlink.io/iLtUuarTCuvRjEgXgcgyWAhDHPsuNRkZvtFiCP_iMy4jgH6YOlGCXNmw8SuG6Zvu4zYcPls9pyDBnZGdqHEd5g.png">(view full size)</a> |<a href="https://rockylinux-gnv9d3fdu-resforg.vercel.app/faq">Screenshot of /faq</a><br /><sup><a href="https://images.weserv.nl?url=https://iad.microlink.io/BpgbTjGJhxViSbWS3lD4T4Pg7LEtuUfPA5FKgG1kJGPpg123i-UMONwLGaj0-iuXOG8f8-l09oACd2Q43UyZOg.png">(view full size)</a> |



| [**/**](https://rockylinux-gnv9d3fdu-resforg.vercel.app/) | [**/partners**](https://rockylinux-gnv9d3fdu-resforg.vercel.app/partners) |
|:-:|:-:|
|<a href="https://rockylinux-gnv9d3fdu-resforg.vercel.app/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://iad.microlink.io/OBzuxSMogQ7PoCRMWNwmP2JkLkIM6nkr17SVLGNsN_UkDNlIJ5XSRR35OmJ9io0dii9b-vKeDGG5ow19GzcoxQ.png">(view full size)</a> |<a href="https://rockylinux-gnv9d3fdu-resforg.vercel.app/partners">Screenshot of /partners</a><br /><sup><a href="https://images.weserv.nl?url=https://iad.microlink.io/_GqWvsr18mI-tLwYE3jgfKfk1vChSMQ5N4x87rgl8fSfXRJ-Lu4K0s3gtCx7jdNLCk9F1aIijyNSuqx7OXarcQ.png">(view full size)</a> |



| [**/sponsors**](https://rockylinux-gnv9d3fdu-resforg.vercel.app/sponsors) |
|:-:|
|<a href="https://rockylinux-gnv9d3fdu-resforg.vercel.app/sponsors">Screenshot of /sponsors</a><br /><sup><a href="https://images.weserv.nl?url=https://iad.microlink.io/Jj2Fjc-GtjcLd4uJ02a01AZ2P4RbgD2WOTmeLHsTzt39h11olYJHk8MIc5bZkLqsqaTvCZzwnN6k4dzrTMWKTQ.png">(view full size)</a> |







Commit cce412c34f95d7dfdd156ba5cdcb57fd5c570b58 (https://rockylinux-gnv9d3fdu-resforg.vercel.app).

**Comment by hbjydev** [comment]
This button needs to be added to the new translation set too

**Comment by FoggyMtnDrifter** [comment]
@hbjydev Which language is that? I'm not able to replicate that.

**Comment by FoggyMtnDrifter** [comment]
Are you looking at the deployment for this PR?

**Comment by hbjydev** [comment]
@ressonix Yep - just checked again on the latest build, and I used Nederlands. Issue's still there

**Comment by FoggyMtnDrifter** [comment]
Okay, looks like it's just that language specifically, all others seem to work fine @hbjydev.

**Comment by hbjydev** [comment]
> Okay, looks like it's just that language specifically, all others seem to work fine @hbjydev.

Ah, so does this not fix the links.* display issue?

**Comment by FoggyMtnDrifter** [comment]
It does fix it, it just looks like that language was left out. I'm updating it now @hbjydev

**Comment by ofaruksahintr** [comment]
@ressonix

locales/nl/index.json:23 --> "links:" to "links"
locales/pt-pt/index.json:23 --> "links:" to "links"

**Comment by FoggyMtnDrifter** [comment]
> @ressonix
> 
> locales/nl/index.json:23 --> "links:" to "links"
> locales/pt-pt/index.json:23 --> "links:" to "links"

Yep, I'm on it. Thanks!

**Comment by FoggyMtnDrifter** [comment]
Should be good to go now @hbjydev

File: src/i18n/locales/en/index.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "faq": {
+    "a": {
+      "come-in": "Rocky Linux aims to function as a downstream build as CentOS had done previously, building releases after they have been added by the upstream vendor, not before.",
+      "downstream-partner-direction": "The CentOS project recently announced a shift in strategy for CentOS. Whereas previously CentOS existed as a downstream build of its upstream vendor (it receives patches and updates after the upstream vendor does), it will be shifting to an upstream build (testing patches and updates before inclusion in the upstream vendor). Additionally, support for CentOS Linux 8 has been cut short, from May 31, 2029 to December 31, 2021."
+    },
+    "q": {
+      "come-in": "So where does Rocky Linux come in?",
+      "downstream-partner-direction": "What do you mean, \"its downstream partner has shifted direction?\""
```
Comment (SergeCroise) [inline src/i18n/locales/en/index.json:9]: Ponctuation:
...has shifted direction\\"?"

File: src/i18n/locales/es/index.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+  "faq": {
+    "a": {
+      "come-in": "Rocky Linux apunta a funcionar como una compilación descendente como lo había hecho CentOS anteriormente, construyendo versiones después de que hayan sido agregadas por el proveedor ascendente, no antes.",
+      "downstream-partner-direction": "El proyecto CentOS anunció recientemente un cambio en la estrategia de CentOS. Mientras que anteriormente CentOS existía como una compilación descendente de su proveedor ascendente (recibe parches y actualizaciones después de que lo hace el proveedor ascendente), se cambiará a una compilación ascendente (probando parches y actualizaciones antes de su inclusión en el proveedor ascendente). Además, el soporte para CentOS Linux 8 se ha interrumpido, cambiando el final del soporte del 31 de mayo de 2029 al 31 de diciembre de 2021."
+    },
+    "q": {
+      "come-in": "Entonces, ¿dónde entra Rocky Linux?",
+      "downstream-partner-direction": "¿Qué significa que, \"su socio anterior haya cambiado de dirección?\""
```
Comment (SergeCroise) [inline src/i18n/locales/es/index.json:9]: Ponctuation:
...haya cambiado de dirección\\"?"

**Comment by hbjydev** [comment]
@SergeCroise That's to escape that final " in each line - so we can have it as a quoted question on the site :)

**Review by hbjydev — APPROVED** [review]

**Pull request merged**